docs/feature_flags.md

```diff
@@ -95,58 +95,6 @@ the corresponding classic histogram, with the notable exception of exemplars,
 which are always ingested. To keep the classic histograms as well, enable
 `scrape_classic_histograms` in the scrape job.
 
-_Note about the format of `le` and `quantile` label values:_
```

Comment:
maybe, in addition to the release notes and migration guide, we could mention the normalization in https://prometheus.io/docs/concepts/data_model/?

Reply:
We have to keep in mind that all the instructions here, being deleted in this PR, should make it into the migration guide. /cc @jan--f

Reply:
(I'll open the needed PR against `prometheus/docs`)